package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "opentelemetry-base",
+  "version": "0.10.2",
+  "description": "OpenTelemetry is a distributed tracing and stats collection framework.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "scripts": {
+    "bench": "node benchmark",
+    "clean": "lerna run clean",
+    "postinstall": "npm run bootstrap",
+    "precompile": "tsc --version",
+    "version:update": "lerna run version:update",
+    "compile": "lerna run compile",
+    "test": "lerna run test",
+    "test:browser": "lerna run test:browser",
+    "bootstrap": "lerna bootstrap",
+    "bump": "lerna publish",
+    "codecov": "lerna run codecov",
+    "codecov:browser": "lerna run codecov:browser",
+    "changelog": "lerna-changelog",
+    "predocs-test": "npm run docs",
+    "docs-test": "lerna run docs-test",
+    "docs": "lerna run docs",
+    "docs-deploy": "gh-pages --dist packages/opentelemetry-api/docs/out",
+    "lint": "lerna run lint",
+    "lint:fix": "lerna run lint:fix",
+    "lint:examples": "eslint ./examples/**/*.js",
+    "lint:examples:fix": "eslint ./examples/**/*.js --fix",
+    "lint:markdown": "./node_modules/.bin/markdownlint $(git ls-files '*.md') -i ./CHANGELOG.md",
+    "lint:markdown:fix": "./node_modules/.bin/markdownlint $(git ls-files '*.md') -i ./CHANGELOG.md --fix"
+  },
+  "repository": "open-telemetry/opentelemetry-js",
```

Comment:
Since when I build this repository, I mainly paid attention to the subdirectories of each component, and to ensure all the component pass the test, some configuration I just directly borrowed from opentelemetry-js repo.
If you find any obvious mistake like this, please feel free to point out, I will modify. Thank you so much!